projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -249,6 +244,23 @@ function JetpackContactFormEdit( { name, attributes, setAttributes, clientId, cl
 						</>
 					) }
 				</InspectorControls>
+				<InspectorAdvancedControls>
+					<TextControl
+						label={ __( 'Accessible name', 'jetpack-forms' ) }
+						value={ formTitle }
+						placeholder={ postTitle }
+						onChange={ value => setAttributes( { formTitle: value } ) }
+						help={ __(
+							'Add an accessible name to help people using assistive technology identify the form. Defaults to page or post title.',
+							'jetpack-forms'
+						) }
+						__nextHasNoMarginBottom={ true }
+						__next40pxDefaultSize={ true }
+					/>
+					<ExternalLink href="https://developer.mozilla.org/docs/Glossary/Accessible_name">
```

Comment:
The link will redirect to translated versions when they're available:

<img width="227" alt="Screenshot 2025-02-27 at 17 36 35" src="https://github.com/user-attachments/assets/0b71fc50-98c7-42c6-bcc7-252540df71be" />